packages/app/src/systems/Network/pages/Networks/Networks.tsx

```diff
@@ -1,38 +1,58 @@
-import { Button, Icon } from '@fuel-ui/react';
-import { useNavigate } from 'react-router-dom';
+import { cssObj } from '@fuel-ui/css';
+import { Button, Dialog, Icon, IconButton } from '@fuel-ui/react';
 
-import { Layout, Pages } from '~/systems/Core';
-import { NetworkList, NetworkScreen, useNetworks } from '~/systems/Network';
+import { coreStyles } from '~/systems/Core/styles';
+import { NetworkList, useNetworks } from '~/systems/Network';
 
-export function Networks() {
-  const navigate = useNavigate();
-  const { networks, isLoading, handlers } = useNetworks({
-    type: NetworkScreen.list,
-  });
+export const Networks = () => {
+  const { networks, handlers } = useNetworks();
 
   return (
-    <Layout title="Networks" isLoading={isLoading}>
-      <Layout.TopBar onBack={() => navigate(Pages.wallet())} />
-      <Layout.Content>
-        {networks && (
-          <NetworkList
-            networks={networks}
-            onUpdate={handlers.goToUpdate}
-            onPress={handlers.selectNetwork}
-            {...(networks?.length > 1 && { onRemove: handlers.removeNetwork })}
-          />
-        )}
-      </Layout.Content>
-      <Layout.BottomBar>
+    <>
+      <Dialog.Heading>
+        Networks
+        <IconButton
+          data-action="closed"
+          variant="link"
+          icon={<Icon icon="X" color="gray8" />}
+          aria-label="Close networks modal"
+          onPress={handlers.closeDialog}
+        />
+      </Dialog.Heading>
+      <Dialog.Description
+        as="div"
+        css={styles.description}
+        data-has-scroll={Boolean((networks || []).length >= 6)}
```

Comment:
setting `data-has-scroll` to increase padding could be removed, and apply the style always instead.

defining scroll boolean based on networks length is fragile because any changes in networkItem element will change height and the dev would need to come back here to change length to scroll...